checks/mysql

```diff
@@ -250,3 +248,171 @@ check_info['mysql.connections'] = {
     "group": "mysql_connections",
     "has_perfdata": True,
 }
+
+#.
+#   .--Galera Sync Status--------------------------------------------------.
+#   |         ____       _                  ____                           |
+#   |        / ___| __ _| | ___ _ __ __ _  / ___| _   _ _ __   ___         |
+#   |       | |  _ / _` | |/ _ \ '__/ _` | \___ \| | | | '_ \ / __|        |
+#   |       | |_| | (_| | |  __/ | | (_| |  ___) | |_| | | | | (__         |
+#   |        \____|\__,_|_|\___|_|  \__,_| |____/ \__, |_| |_|\___|        |
+#   |                                             |___/                    |
+#   |                    ____  _        _                                  |
+#   |                   / ___|| |_ __ _| |_ _   _ ___                      |
+#   |                   \___ \| __/ _` | __| | | / __|                     |
+#   |                    ___) | || (_| | |_| |_| \__ \                     |
+#   |                   |____/ \__\__,_|\__|\__,_|___/                     |
+#   |                                                                      |
+#   +----------------------------------------------------------------------+
+
+def inventory_mysql_galerasync(parsed):
+    for instance, values in parsed.iteritems():
+        if values.get('wsrep_provider', 'none') != 'none' and u'wsrep_local_state_comment' in values:
+            yield instance, {}
+
+@get_parsed_item_data
+def check_mysql_galerasync(item, _no_params, values):
+    if u'wsrep_local_state_comment' in values:
+        if values[u'wsrep_local_state_comment'] == 'Synced':
+            state = 0
+        else:
+            state = 2
+        return (state, 'WSREP local state comment: %s' % values[u'wsrep_local_state_comment'])
+
+check_info['mysql.galerasync'] = {
+    "inventory_function"      : inventory_mysql_galerasync,
+    "check_function"          : check_mysql_galerasync,
+    "service_description"     : "MySQL Galera Sync %s",
+    "has_perfdata"            : False,
```

Comment:
that's the default, can be removed

Reply:
see commit 8437746bad7d4c8d697695b44f24c64185814091